code/API_definitions/Discovery/simple_edge_discovery.yaml

```diff
@@ -7,7 +7,7 @@ info:
     ---
     # Summary
 
-    The Simple Edge Discovery API returns the name of the closest operator MEC platform to a particular user device. 
+    The Simple Edge Discovery API returns the name of the closest operator MEC platform to a particular user device.
```

Comment:
Should it be "platforms"? Singular to Plural?

Reply:
@gunjald I'm not changing the content or meaning of the API.
I'm just applying linting and spelling. These issues you comment need a dedicated PR.